lib/components/dropdown.vue

```diff
@@ -76,6 +67,11 @@
                 type: String,
                 default: null
             }
+        },
+        methods:{
+            log() {
```

Comment:
That was a typo for local test only 😆 Burned in 15b8674e6c1e6ace1a7d564c5033b9b176aa831c